.github/workflows/ci.yml

```diff
@@ -46,6 +46,12 @@ jobs:
         - platform: { name: Windows VS2022, os: windows-2022 }
           config: { name: Unity, flags: -DSFML_USE_MESA3D=TRUE -DBUILD_SHARED_LIBS=TRUE -DCMAKE_UNITY_BUILD=ON }
           type: { name: Debug, flags: -DCMAKE_BUILD_TYPE=Debug -DSFML_ENABLE_COVERAGE=TRUE }
+        - platform: { name: Windows VS2022, os: windows-2022 }
+          config: { name: OpenGL ES, flags: -DSFML_USE_MESA3D=TRUE -DBUILD_SHARED_LIBS=TRUE -DSFML_OPENGL_ES=ON }
```

Comment:
You can omit `-DBUILD_SHARED_LIBS=TRUE` since they're on by default. Are shared libs truly a requirement anyways?

Reply:
I just took this from the existing configurations. It wouldn't make sense to declare it sometimes and sometimes not, either we are explicit or we aren't.